docs/glossary.md

```diff
@@ -322,6 +381,19 @@ A [simulation network](advanced/simnet.md) with very low difficulty, such that a
 
 A wallet mode in which only blocks related to addresses owned by the wallet are downloaded. When not in SPV mode, wallets must download all blocks in the blockchain to verify transactions (fully validating mode). SPV mode allows wallets to operate with less stringent hardware requirements and load significantly faster. SPV wallets cannot vote, but can purchase tickets and allocate voting rights to a [Voting Service Provider (VSP)](#voting-service-provider).
 
+#### Smart contract
+
+A smart contract is a self-executing contract with the terms of the agreement
+between buyer and seller being directly written into lines of code.
+
+Once a smart contract has been written and broadcast to the Decred network, the
```

Comment:
I think we could shorten this to make it more concise like the other definitions. Also, I think we need to address the general confusion around the definition of smart contracts. Many people think smart contracts are only turning complete (or more so) contracts like Ethereum has. We don't want people thinking Decred is a "smart contract platform", when it only (for now) has smart contracts in the more limited definition shared with Bitcoin.

Reply:
Added a line

> The scripting language used to create Decred smart contracts is not [Turing complete](https://en.wikipedia.org/wiki/Turing_completeness).